io.github.Cookiiieee.WSelector.json

```diff
@@ -0,0 +1,71 @@
+{
+    "app-id": "io.github.Cookiiieee.WSelector",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "48",
+    "sdk": "org.gnome.Sdk",
+    "command": "wselector",
+    "build-options": {
+        "build-args": [
+            "--share=network"
+        ]
+    },
+    "modules": [
+        {
+            "name": "python-deps",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 -m pip install --prefix=${FLATPAK_DEST} --no-binary :all: urllib3 charset-normalizer certifi idna requests beautifulsoup4 psutil"
+            ]
+        },
+        {
+            "name": "wselector",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/Cookiiieee/WSelector.git",
+                    "branch": "main"
+                }
+            ],
+            "build-commands": [
+                "PYVER=$(python3 -c 'import sys; print(f\"{sys.version_info.major}.{sys.version_info.minor}\")')",
+                "install -Dm644 wselector.desktop ${FLATPAK_DEST}/share/applications/io.github.Cookiiieee.WSelector.desktop",
+                "install -Dm644 io.github.Cookiiieee.WSelector.metainfo.xml ${FLATPAK_DEST}/share/metainfo/io.github.Cookiiieee.WSelector.metainfo.xml",
+                "mkdir -p ${FLATPAK_DEST}/lib/python${PYVER}/site-packages",
+                "cp -r wselector ${FLATPAK_DEST}/lib/python${PYVER}/site-packages/",
+                "chmod +x install-icons.sh",
+                "./install-icons.sh ${FLATPAK_DEST}",
+                "update-desktop-database ${FLATPAK_DEST}/share/applications",
+                "install -D /dev/null ${FLATPAK_DEST}/bin/wselector",
+                "echo '#!/bin/sh' > ${FLATPAK_DEST}/bin/wselector",
+                "echo 'export PYTHONPATH=/app/lib/python${PYVER}/site-packages' >> ${FLATPAK_DEST}/bin/wselector",
+                "echo 'exec /usr/bin/python3 -m wselector \"$@\"' >> ${FLATPAK_DEST}/bin/wselector",
+                "chmod +x ${FLATPAK_DEST}/bin/wselector"
+            ]
+        }
+    ],
+    "finish-args": [
+        "--share=network",
+        "--socket=wayland",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=session-bus",
+        "--filesystem=xdg-pictures/WSelector:create",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.freedesktop.portal.settings",
+        "--talk-name=org.freedesktop.FileManager1",
+        "--talk-name=org.gtk.Actions"
```

Comment:
I'm not even sure what this is. There is no dbus with the name `org.gtk.Actions`